pages/property-filter/table.data.ts

```diff
@@ -13,8 +13,8 @@ export interface TableItem {
   publicdns?: string;
   ipv4publicip?: string;
   securitygroup?: string;
-  launchdate?: string;
-  lasteventat?: string;
+  launchdate?: Date;
```

Comment:
We expected the items date value to be of type Date.

This assumption is used in the collection-hooks: https://github.com/cloudscape-design/collection-hooks/blob/main/src/date-utils/compare-dates.ts#L8.